hackathon_site/dashboard/frontend/src/components/general/Navbar/Navbar.test.tsx

```diff
@@ -2,27 +2,28 @@ import React from "react";
 
 import { fireEvent, render } from "testing/utils";
 
-import Navbar, { UnconnectedNavbar } from "./Navbar";
-import { styles } from "./Navbar.module.scss";
-import { cartQuantity } from "testing/mockData";
+import { UnconnectedNavbar } from "./Navbar";
+import styles from "./Navbar.module.scss";
 
 describe("<Navbar />", () => {
+    const handleLogoutSpy = jest.fn();
+
     const pagesAndPaths = [
         ["Dashboard", "/"],
         ["Orders", "/orders"],
         ["Teams", "/teams"],
         ["Reports", "/reports"],
         ["Inventory", "/inventory"],
-        [`Cart (${cartQuantity})`, "/cart"],
+        [`Cart (0)`, "/cart"],
     ];
 
     pagesAndPaths.map(([label, path]) => {
         it(`Adds the active class to ${label} when on ${path}`, () => {
             const { getByText, container } = render(
-                <Navbar cartQuantity={cartQuantity} pathname={path} />
+                <UnconnectedNavbar pathname={path} logout={handleLogoutSpy} />
             );
 
-            expect(getByText(label).closest("button").className).toMatch(
+            expect(getByText(label).closest("button")!.className).toMatch(
```

Comment:
Wait this doesnt work without the ! ?

Reply:
it says object is possible null

Reply:
Could you use ? Instead of ! 
I haven't seen ! used before in this way what does it do?

Reply:
Oh I was thinking the ! worked the same as the ?, I guess I'll just change it.